frontend/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -58,20 +65,22 @@ const TooltipWrapper = ({
       <div className={elementClassNames} data-tooltip-id={tipId}>
         {children}
       </div>
-      <ReactTooltip5
-        className={tipClassNames}
-        id={tipId}
-        delayShow={isDelayed ? 500 : undefined}
-        delayHide={isDelayed ? 500 : undefined}
-        noArrow
-        place={position}
-        opacity={1}
-        disableStyleInjection
-        clickable={clickable}
-        offset={5}
-      >
-        {tipContent}
-      </ReactTooltip5>
+      {!disableTooltip && (
+        <ReactTooltip5
+          className={tipClassNames}
+          id={tipId}
+          delayShow={isDelayed ? 500 : undefined}
+          delayHide={isDelayed ? 500 : undefined}
+          noArrow
```

Comment:
Not for this PR, but it looks like we ought to be able to support a wrapper that allows us to conditionally set [arrow](https://react-tooltip.com/docs/examples/basic-examples).